api/genomics/wdl/IngestSyntheticVcfWf.wdl

```diff
@@ -236,19 +187,21 @@ task CreateImportTsvs {
         --probe-info-file ~{probe_info_file} \
         --ref-version 37
 
-      mv *.tsv output_tsvs
+      for i in 1 2 3;
```

Comment:
This seems potentially flaky. What was wrong with the previous code?

Reply:
`gsutil cp` would sometimes fail and kill the task. This adds a 3x retry loop.

Reply:
does `gsutil -m cp` help that at all? I'd prefer that be tried first rather than a retry loop

Reply:
Hmm, I feel like it wouldn't since the error I got was a service unavailable response (503) from Google. The general consensus for resolution was to just try again after you get that response since 503s are typically intermittent.

That said, I can add the `-m` flag to this command since it wouldn't hurt.

Unfortunately, I can't really test it since it fails so rarely that I need to run a large job to reliably trigger the failure mode and that would cost a pretty penny.

Reply:
Sorry for the late reply. `outdir` is typically a GCS bucket but I think it can support directories since Cromwell will sort that out for you automatically.

I agree about the overhead with pulling it out into another WDL step. It would be nice to utilize Cromwell's built in retries but there's some time/$ costs associated with pushing the files around.